lab-integration/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "env": {
+        "node": true,
+        "es6": true
+    },
+    "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:prettier/recommended"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
```

Comment:
Option module allow **import/export** usage... and **script** report as sintax error the **import/export statments**. I don't how was possible nothing happenned when you remove it.  Then **script** is the default option and with this configuration the behavior will be ESLint throwing parsing errors because it treats our code as a script that don't allow **import/export** statments.